examples/calibration/plot_calibration_curve.py

```diff
@@ -111,20 +112,24 @@ def plot_calibration_curve(est, name, fig_index):
         ax1.plot(mean_predicted_value, fraction_of_positives, "s-",
                  label="%s (%1.3f)" % (name, clf_score))
 
-        ax2.hist(prob_pos, range=(0, 1), bins=10, label=name,
-                 histtype="step", lw=2)
+        pd.Series(prob_pos).plot(kind='density', label=name, lw=2, ax=ax2)
```

Comment:
You could also get a density plot with a matplotlib hist plot and `density=True`.